docs/Model_Assumptions.rst

```diff
@@ -287,6 +287,89 @@ For example, the `degree of sector coupling` will reach its maximum, when the sy
 
 In the MVS, there is no distinction between energy carriers and energy vector. For `Electricity` of the `Electricity` vector this may be self-explanatory. However, the energy carriers of the `Heat` vector can have different technical characteristics: A fluid on different temperature levels. As the MVS measures the energy content of a flow in kWh(thermal) however, this distinction is only relevant for the end user to be aware of, as two assets that have different energy carriers as an output should not be connected to one and the same bus if a detailed analysis is expected. An example of this would be, that a system where the output of the diesel boiler as well as the output of a solar thermal panel are connected to the same bus, eventhough they can not both supply the same kind of heat demands (radiator vs. floor heating).  This, however, is something that the end-user has to be aware of themselves, eg. by defining self-explanatory labels.
 
+CO2 Emission factors
+----------------------------
+In order to optimise the energy system with minimum CO2 emissions, it is important to calculate CO2 emission per unit of fuel consumption.
+
+Following emission factors for energy carriers are defined:
+
+.. _table_CO2_emissions_energyCarriers:
+
+.. list-table:: CO2 Emission factors: Kg of CO2 equivalent per unit of fuel consumption
+   :widths: 50 25 25 25
+   :header-rows: 1
+
+   * - Energy carrier
```

Comment:
Could you change the order of "Unit" and "Value" in this table please, so that it has the same order as the second table?

Reply:
I changed the order of "Unit" and "Value".